plugins/versions.properties

```diff
@@ -11,14 +11,16 @@ version.junit=5.7.0
 
 version.kotest=4.3.1
 
-version.kotlin=1.4.20
+version.kotlin=1.3.72
 
-version.kotlinpoet=1.7.2
+version.kotlinpoet=1.6.0
 
-version.kotlinx.coroutines=1.3.9
+version.kotlinx.coroutines=1.2.1
```

Comment:
I think we should use version 1.3.8, the latest Kotlin 1.3 compatible release IIRC.